addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1604,7 +1610,7 @@ body.editor_enable.editor_has_snippets {
                 }
             }
 
-            &[data-name="transparent_grayscale"], &[data-name="theme"], &[data-name="reset"] {
+            &[data-name="transparent_grayscale"], &[data-name="theme"], &[data-name="reset"], &[data-name="custom"] {
```

Comment:
It's either something new (what is this?) or something to fix in a stable version, not master?

Reply:
In stable, custom colors are displayed alongside the theme colors, in the theme div. I created a new div to display the custom colors separately from the theme colors, but the elements of this new div need the same style to display transparency correctly